### PR TITLE
Code cleanup and introducing updater module

### PR DESCRIPTION
**Pull request by HitaloSama**
I did a general cleaning of the code using PyCharm and adapted the [Watzon's tg_userbot updater module](https://github.com/watzon/tg_userbot).

### CODE REVIEW

**Comment by corsicanu** [comment]
The only use i see for updater is on heroku, on a vps and manually started service bot can't start itself after pulling the changes.

File: tg_userbot/modules/updater.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    await ups.client.disconnect()
+    # Spin a new instance of bot
+    execl(sys.executable, sys.executable, *sys.argv)
+    # Shut the existing one down
+    exit()
+
+
+CMD_HELP.update(
+    {"updater": "`.update`: Check if the main repository has any updates and show changelog if so.\
+        \n\n`.update now`: Update Paperplane if there are any updates available."})
```
Comment (corsicanu) [inline tg_userbot/modules/updater.py:109]: This is not paperplane tho

File: tg_userbot/modules/updater.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    return
+
+
+@register(outgoing=True, pattern=r"^\.update(?: |$)(.*)")
+@errors_handler
+async def upstream(ups):
+    """For .update command, check if the bot is up to date, update if specified"""
+    await ups.edit("`Checking for updates, please wait....`")
+    conf = ups.pattern_match.group(1)
+    off_repo = 'https://github.com/nunopenim/tguserbot.git'
```
Comment (corsicanu) [inline tg_userbot/modules/updater.py:36]: please don't hardcode the upstream, people migh use own fork, let it be env variable

Comment (HitaloSama) [reply tg_userbot/modules/updater.py:36]: @corsicanu I'll follow your tips, until then I'll leave this PR closed in how much I correct some things.

**Pull request closed without merging**